tests/system/runtimes/test_databricks.py

```diff
@@ -227,3 +274,50 @@ def test_missing_code_run(self):
                 "Databricks function must be provided with user code"
                 in bad_request_error.value
             )
+
+    def test_abort_task(self):
+        #  in order to clean up any active runs:
+        if self.project.list_runs(state="running"):
+            self.project = mlrun.projects.new_project(self.project_name, overwrite=True)
+        sleep_code = """
+
+import time
+
+def handler(**kwargs):
+    time.sleep(1000) # very long sleep
+
+"""
+        function_ref = FunctionReference(
+            kind="databricks",
+            code=sleep_code,
+            image="mlrun/mlrun",
+            name="databricks_abort_test",
+        )
+        function = function_ref.to_function()
+        self._add_databricks_env(function=function)
+        test_run_uid = uuid.uuid4()
+        databricks_run_name = f"databricks_abort_test_{test_run_uid}"
+        params = {"task_parameters": {"databricks_run_name": databricks_run_name}}
+        run_thread = threading.Thread(
+            target=lambda: function.run(project=self.project_name, params=params)
+        )
+        run_thread.start()
+        # wait for databricks to run the function.
+        time.sleep(10)
+        workspace = WorkspaceClient()
+        run = self._get_active_run_by_name(
+            workspace=workspace, run_name=databricks_run_name
+        )
+        assert run.state.life_cycle_state in (
+            RunLifeCycleState.PENDING,
+            RunLifeCycleState.RUNNING,
+        )
+        self._abort_run()
+        run = workspace.jobs.get_run(run_id=run.run_id)
+        # wait for databricks to update the status.
+        time.sleep(5)
```

Comment:
Pretty sure you need to sleep before `get_run` for the sleep to have any effect (other than slowing the test down).